core/services/helpers_test.go

```diff
@@ -0,0 +1,71 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func ExportedSetCheckerFactory(fm FluxMonitor, fac DeviationCheckerFactory) {
+	impl := fm.(*concreteFluxMonitor)
+	impl.checkerFactory = fac
+}
+
+func mustReadFile(t testing.TB, file string) string {
+	t.Helper()
+
+	content, err := ioutil.ReadFile(file)
+	require.NoError(t, err)
+	return string(content)
+}
+
+type fixedFetcher struct {
+	price float64
+}
+
+func newFixedPricedFetcher(price float64) *fixedFetcher {
+	return &fixedFetcher{price: price}
+}
+
```

Comment:
alot of this code looks like its strictly in support of the flux monitor, maybe giving it a more distinct name or moving the code to something else would help.

A suggestion would be flux_monitor_helpers_test.go ?